tools/development/register-boot2docker.sh

```diff
@@ -11,10 +10,14 @@ fi
 # This is just here to make sure your environment is sane
 docker info
 
-CONSOLE_ARGS=""
-if [ -t 1 ]; then
```

Comment:
These allow interrupting the process while its running. Something ibuildthecloud does... Doesn't look like they are being set/used